docs/index.md

```diff
@@ -1,8 +1,10 @@
-# TAXPASTA Documentation
+# About
```

Comment:
Promoting "About" to the title, I don't quite understand.

Reply:
Otherwise you have a header called taxpasta above the logo which says taxpasta. It's redundant and looked ugly

Reply:
Let me try something, might be able to name the document title via configuration.

Reply:
I attempted that and failed